Breadcrumbs.py

```diff
@@ -108,4 +132,213 @@ def get_points_by_row(row):
               breadcrumbs[index_of_breadcrumb] = breadcrumbs[index_of_breadcrumb][0:length_of_trim]
           break
       number_of_characters_left -= current_length
-    view.set_status('breadcrumbs',separator.join(breadcrumbs+['']))
+  return breadcrumbs
+
+
+def copy(view, text):
+  sublime.set_clipboard(text)
+  sublime.status_message('Breadcrumbs copied to clipboard')
+
+
+class BreadcrumbsEventListener(sublime_plugin.ViewEventListener):
```

Comment:
The class `sublime_plugin.ViewEventListener` is not available in ST2 :(
```
Traceback (most recent call last):
  File ".\sublime_plugin.py", line 62, in reload_plugin
  File ".\Breadcrumbs.py", line 143, in <module>
    class BreadcrumbsEventListener(sublime_plugin.ViewEventListener):
AttributeError: 'module' object has no attribute 'ViewEventListener'
```
I want to have this statusbar feature mostly as a fallback for ST2, so please revert this to the version which used `EventListener` instead. A statusbar which only works in ST3 is not really needed, as in ST3 we now have beautiful phantoms.

I will probably use the `ViewEventListener` version you proposed here to make a ST3-only version which will use toggleable phantoms which follow the cursor, so thanks for investigating this.